### ISSUE TITLE
Support Windows

### ISSUE THREAD

**Issue by moorepants** [conda-forge/cyipopt-feedstock#6]
Note that the ipopt-feedstock now supports Windows, so this can be updated.

https://github.com/conda-forge/ipopt-feedstock/pull/11

**Comment by richardotis** [comment]
Not sure how to support this on conda-forge yet, but I can build these from source locally (in order) with `conda build .`:
https://github.com/richardotis/mumps-feedstock/tree/windows_support
https://github.com/richardotis/ipopt-feedstock/tree/windows_support
https://github.com/richardotis/cyipopt-feedstock/tree/master

You do need msys2 in your list of channels.

**Comment by moorepants** [comment]
The only pertinent step is to get ipopt supported on windows:

https://github.com/conda-forge/ipopt-feedstock/

**Comment by bocklund** [comment]
Seems like this is close to being done, tests are passing and it just needs a final merge https://github.com/conda-forge/ipopt-feedstock/pull/24

**Comment by moorepants** [comment]
IPOPT just got windows support upstream: https://github.com/conda-forge/ipopt-feedstock/pull/47

**Comment by bocklund** [comment]
Awesome! Thanks for all the effort @traversaro

**Comment by moorepants** [comment]
I just pushed cyipopt 0.3.0 which should work on this feedstock. It requires setting the environment variable `IPOPTWINDIR="USECONDAFORGEIPOPT"` and `lapack<3.9.0` (or `blas.pc` and `lapack.pc` need to be patched in here).

**Comment by moorepants** [comment]
Here is what is installed if you set `lapack<3.9.0`.

`lapack.pc`

```
prefix=/home/moorepants/miniconda/envs/cyipopt-dev
libdir=/home/moorepants/miniconda/envs/cyipopt-dev/lib

Name: lapack
Description: FORTRAN reference implementation of LAPACK Linear Algebra PACKage
Version: 3.6.1
URL: http://www.netlib.org/lapack/
Libs: -L/home/moorepants/miniconda/envs/cyipopt-dev/lib -llapack
Requires: blas
```

`blas.pc`

```
prefix=/home/moorepants/miniconda/envs/cyipopt-dev
libdir=/home/moorepants/miniconda/envs/cyipopt-dev/lib

Name: blas
Description: Basic Linear Algebra Subprograms F77 reference implementations
Version: 3.6.1
URL: http://www.netlib.org/blas/
Libs: -L/home/moorepants/miniconda/envs/cyipopt-dev/lib -lblas
Libs.private: -lm
```

We could add these files to this feedstock so it would work with lapack >= 3.9.

**Comment by moorepants** [comment]
Fixed with #26!

**Comment by bocklund** [comment]
Thank you so much, @moorepants!
